sdk/messaging/azservicebus/go.mod

```diff
@@ -25,7 +24,10 @@ require (
 	nhooyr.io/websocket v1.8.6
 )
 
-require golang.org/x/net v0.0.0-20220425223048-2871e0cb64e4
+require (
+	github.com/golang/mock v1.6.0
+	github.com/google/uuid v1.1.1
```

Comment:
Is this an indirect dependency?

Reply:
Ah, I removed it. Sometimes it gets inserted when I'm trying to find the internal/uuid package.